ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,31 @@
-<!--
-  Thanks for your interest in using the App Center SDK for React Native.
+### **Description**
 
-  If you have an issue in React Native platform but have not integrated
```

Comment:
@elamalani IMHO I think we should still keep this clarification about using intercom for support tickets.